.github/workflows/main.yml

```diff
@@ -879,17 +879,17 @@ jobs:
               image: "windows-arm64",
               stockfish: "no",
               stockfish_arg: "ARCH=armv8 COMP=clang",
-              ninja: "no",
+              ninja: "yes",
               ninja_arg: "",
               openssl: "no",
               openssl_arg: "mingw64 no-asm",
               C: "no",
               C_arg: "",
-              C-Plus-Plus: "yes",
+              C-Plus-Plus: "no",
```

Comment:
I am not entirely sure what that implies... is it saying that windows-arm64 does not support C++? :thinking:

Reply:
Honestly, I'm not sure why this happens, but it is the same for other Windows images.

Reply:
Ok :sweat_smile:. But do you compile anything C++ with that image (or any Windows image that has this set to "no")? :grin:

Reply:
For sure windows_arm64 and windows-static-x64 produce correct dll in this project:
https://github.com/java-native/jssc/pull/140

Reply:
`Ç-Plus-Plus` is sometimes a little capricious, [fmt](https://github.com/fmtlib/fmt) and [ninja](https://github.com/ninja-build/ninja) works